coach/src/concept-coach/error-notification.cjsx

```diff
@@ -37,17 +37,20 @@ ErrorNotification = React.createClass
   componentWillUnmount: ->
     api.channel.off 'error', @onError
 
-  onError: ({response, failedData, exception}) ->
-    return if failedData?.stopErrorDisplay # someone else is handling displaying the error
+  onError: (exception) ->
+    {config, response} = exception
+
+    # return if failedData?.stopErrorDisplay # someone else is handling displaying the error
     if exception?
```

Comment:
This type of thing is no longer needed with the new api handler! :)  this error only get triggered if the endpoint specific error handler hasn't handled it.  for coach, these are defined in the routes as `handledErrors`